presto-hive/src/main/java/com/facebook/presto/hive/parquet/ParquetPageSourceFactory.java

```diff
@@ -165,7 +164,7 @@ public ParquetPageSourceFactory(TypeManager typeManager,
             HiveFileContext hiveFileContext,
             Optional<EncryptionInformation> encryptionInformation)
     {
-        if (!PARQUET_SERDE_CLASS_NAMES.contains(storage.getStorageFormat().getSerDe()) || shouldUseRecordReaderFromInputFormat(configuration, storage)) {
```

Comment:
@arunthirupathi this code was already how it was working. So behavior change wise, this is not new. FYI